.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,10 @@
 - Which task/US does it implement?
 - Which issue does it fix?
 
+## Dependencies
+
+Does it depend on `rero-ils-ui#<xx>`'s PRs?
```

Comment:
If we can facilitate the way developer complete the text, why not adding `rero/rero-ils-ui#<xx>` directly?
Moreover, a sentence like : 
```
This PR depends on `rero/rero-ils-ui#<xx>`.
```

This way developer will just delete the line if useless. Or just change `<xx>`. A timesave in fact!